etc/releasing.md

```diff
@@ -204,14 +253,51 @@ pushed.
     stable release branch.
   - Commit and push the symlink change:
     `git commit -am "Update symlink for r1.2.3"`
+   - Switch back to the branch with documentation updates: `git checkout post-release-changes`.
 - Wait a few minutes and verify mongocxx.org has updated.
-- Checkout the master branch. Push the commit containing changes to `etc/` and
-  `docs/`. This may require pushing the commit to a fork of the C++ Driver
-  repository and creating a pull request.
+
+## Update CHANGELOG.md post-release ...
+
+CHANGELOG.md on the `master` branch contains sections for every release. This is intended to ease searching for changes among all releases.
+CHANGELOG.md on a release branch (e.g. `releases/v1.2`) contains entries for patch releases of the minor version number tracked by the release branch (e.g. for 1.2.1, 1.2.2, 1.2.3, etc.), as well as all entries prior to the initial minor release (e.g. before 1.2.0).
+
+### ... on the `master` branch
+
+Check out the `post-release-changes` branch created before.
+
+Ensure `[Unreleased]` is removed from the recently released section. Ensure the contents of the recently released section match the published release notes.
+
+Ensure there are `[Unreleased]` sections for the next minor and patch releases. Example (if `1.2.3` was just released):
+
+```md
+## 1.3.0 [Unreleased]
+<!-- Will contain entries for the next minor release -->
+## 1.2.4 [Unreleased]
+<!-- Will contain entries for the next patch release -->
+## 1.2.3
+<!-- Contains published release notes -->
+```
+
+Commit the change. Create a PR from the `post-release-changes` branch to merge to `master`.
+
```

Comment:
```suggestion
```
(Nearly) all post-release _and_ pre-release changes that end up on the `master` branch will be the same as those on the release branch (regardless of minor or patch release). I believe it would be preferable to push all changes to the release branch (no intermediate PR required) before creating a PR to merge all release-related changes to the `master` branch.

Reply:
Suggestion retracted.